build/LinuxDistrosNativeInstaller.props

```diff
@@ -0,0 +1,28 @@
+<Project>
+  <ItemGroup>
+    <LinuxDistrosNeedNativeInstaller Include="ubuntu.16.04-x64">
+      <DockerFolder>ubuntu.16.04</DockerFolder>
+      <InstallerExtension>deb</InstallerExtension>
+      <RepositoryId>$(ubuntu1604x64RepositoryId)</RepositoryId>
+      <DistroNameInInstaller>ubuntu.16.04-x64</DistroNameInInstaller>
```

Comment:
This field has duplicated logic, I don't know how to fix it. I cannot find a easy way for msbuild to look up a dictionary, or even better a function that map osname, rid, architect to this field.